pandas/_libs/tslibs/period.pyx

```diff
@@ -1841,6 +1841,13 @@ cdef class _Period(PeriodMixin):
         Returns
         -------
         Timestamp
+
+        Examples
+        --------
+        >>> period = pd.Period('2023-1-1', freq='D')
+        >>> period = period.to_timestamp()
```

Comment:
could we use a different name here since it is no longer a period?

Reply:
Done.. 
I also added to the DataFrame.to_timestamp example, to show the use of `freq`.